diff -ur codelite-6.0.1.old/CMakeLists.txt codelite-6.0.1/CMakeLists.txt
--- codelite-6.0.1.old/CMakeLists.txt	2015-01-04 12:40:55.188932549 -0500
+++ codelite-6.0.1/CMakeLists.txt	2015-01-04 12:53:19.874354986 -0500
@@ -127,7 +127,7 @@
 else()
     set(WITH_LLDB 0)
 endif()
-if(ENABLE_LLDB MATCHES 0)
+if(NOT ENABLE_LLDB)
     set(WITH_LLDB 0)
     message("-- Disabling lldb support")
 endif ()
@@ -137,9 +137,9 @@
 ## Locate libssh
 #######################################
 ## Enable SFTP support?
-if(ENABLE_SFTP MATCHES 0)
+if(NOT ENABLE_SFTP)
     set( WITH_SFTP 0 )
-endif (ENABLE_SFTP MATCHES 0)
+endif ()
 unset(ENABLE_SFTP CACHE)
 
 if ( WITH_SFTP )
@@ -169,9 +169,9 @@
 endif ( WITH_SFTP )
 
 ## enable clang support?
-if(ENABLE_CLANG MATCHES 0)
+if(NOT ENABLE_CLANG)
   set( USE_CLANG 0 )
-endif (ENABLE_CLANG MATCHES 0)
+endif ()
 unset(ENABLE_CLANG CACHE)
 
 if (AUTOGEN_REVISION MATCHES 0)
@@ -191,7 +191,7 @@
 unset(WITH_WXC CACHE)
 
 ## package the wx libs?
-if (COPY_WX_LIBS MATCHES 1)
+if (COPY_WX_LIBS)
   set( CL_COPY_WX_LIBS 1 )
 endif()
 unset(COPY_WX_LIBS CACHE)
@@ -241,7 +241,7 @@
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O0")        ## No optimization, debug info
 
     ## In debug, only add NDEBUG if the user says so
-    if (PREVENT_WX_ASSERTS MATCHES 1)
+    if (PREVENT_WX_ASSERTS)
     message("-- Adding -DNDEBUG to definitions")
         add_definitions(-DNDEBUG)
     endif()
@@ -271,7 +271,7 @@
     endif(CMAKE_COMPILER_IS_GNUCXX)
 
     ## In release, add NDEBUG unless explicitly told not to
-    if (NOT PREVENT_WX_ASSERTS MATCHES 0)
+    if (PREVENT_WX_ASSERTS)
     message("-- Adding -DNDEBUG to definitions")
         add_definitions(-DNDEBUG)
     endif()
